config/production.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+	entities: {
+		hostess: '5c4955c210728692df56a6a4',
```

Comment:
Let's move this from production to develop and run this in heroku as:
`NODE_ENV=develop node start`